### PR TITLE
Fix markdown editors bug #811 using asset gem.

### PR DESCRIPTION
**Pull request by nishanthvijayan**
As suggested by @hennevogel I've replaced [bootstrap-markdown.js](https://github.com/toopay/bootstrap-markdown) , [to-markdown.js](https://github.com/domchristie/to-markdown) and markdown.js with ther corresponding asset gems thereby upgrading bootstrap-markdown as well.This fixes issue #811

### CODE REVIEW

**Comment by sonalkr132** [comment]
I think [markdown.js](https://github.com/openSUSE/osem/blob/master/vendor/assets/javascripts/markdown.js) and [to-markdown.js](https://github.com/openSUSE/osem/blob/master/vendor/assets/javascripts/to-markdown.js)  files can be removed as well.

**Comment by nishanthvijayan** [comment]
I did remove them from vendor/assets/javascripts/  and instead included  gems 'rails-assets-to-markdown' and 'rails-assets-markdown'

**Comment by sonalkr132** [comment]
Yeah. Sorry!

**Comment by nishanthvijayan** [comment]
On a related note, [rails-assets guidelines](https://rails-assets.org/) states that it requires bundler version to be  >= 1.8.4 . Shouldn't this be enforced in the Gemfile ?

**Comment by hennevogel** [comment]
@nishanthvijayan it should

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+# rails-assets requires >= 1.8.4
+gem 'bundler', '>= 1.8.4'
```
Comment (hennevogel) [inline Gemfile]: This will only install bundler in the bundle but not force the version it's built with :-)

Comment (hennevogel) [inline Gemfile]: Something like this in the Gemfile sounds about right:

if Gem::Version.new(Bundler::VERSION) < Gem::Version.new('1.8.4')
  abort "Bundler version >= 1.8.4 is required"
end

Comment (nishanthvijayan) [inline Gemfile]: Aah true..
But they have explicitly mentioned to include:
`gem 'bundler', '>= 1.8.4'`
in the Gemfile

screenshot from 2016-03-08 20 32 09

Comment (hennevogel) [inline Gemfile]: Well that doesn't make it right :-)

Comment (nishanthvijayan) [inline Gemfile]: So can I make another commit with what you suggested?

Comment (hennevogel) [inline Gemfile]: Please do so :-)

**Comment by hennevogel** [comment]
Wonderful. Another cruft cleaned up. Thank you!
